src/components/Users/User.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import userPhoto from "../../assets/images/user.png";
+import { NavLink } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+
+const User = ({ users, followingInProgress, unfollow, follow }) => {
+  return (
+    <div>
+      {users.map((user) => (
+        <StyledUserItems key={user.id}>
+          <span>
```

Comment:
It's a bad idea to wrap anything in 'span' other than text. If you need just horizontal direction - use styled components like this: 
`const MyHorizontalStyledComponent = styled.div'display: flex; flex-direction: row'`